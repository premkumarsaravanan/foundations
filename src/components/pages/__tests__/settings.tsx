import * as React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Settings from '../settings'

jest.mock('../../../core/store')

describe('Settings', () => {
  it('should match a snapshot', () => {
    expect(toJson(shallow(<Settings />))).toMatchSnapshot()
  })
})
