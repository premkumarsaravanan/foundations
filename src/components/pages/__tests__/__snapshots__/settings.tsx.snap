// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings should match a snapshot 1`] = `
<FlexContainerBasic
  flexColumn={true}
  hasPadding={true}
>
  <Content>
    <FlexContainerResponsive
      flexColumn={true}
      hasBackground={true}
      hasPadding={true}
    >
      <H3>
        Settings
      </H3>
      <H4>
        Payment Info
      </H4>
    </FlexContainerResponsive>
  </Content>
</FlexContainerBasic>
`;
